### PR TITLE
Add Jakarta EL 5.0 Plan Review Ballot Results

### PR DESCRIPTION
**Pull request by Pandrex247**
Ballot results for https://github.com/jakartaee/specifications/pull/363

Signed-off-by: Andrew Pielage <andrew.pielage@payara.fish>

### CODE REVIEW

**Review by ivargrimstad — APPROVED** [review]

File: expression-language/5.0/_index.md [hunk 1]
```diff
@@ -44,21 +44,23 @@ The following TCK updates will be required:
...
-|Oracle                 |         |     |          |
-|Payara                 |         |     |          |
-|Red Hat                |         |     |          |
-|Tomitribe              |         |     |          |
-|EE4J PMC               |         |     |          |
-|Participant Members    |         |     |          |
-|Committer Members      |         |     |          |
-
- This ballot was conducted on the public e-mail list [jakarta.ee-spec@eclipse.org](). This ballot thread begins [here](TBD).
+The Plan Review Specification Committee Ballot concluded successfully on 12/05/2021 with the following results.
```
Comment (kwsutter) [inline expression-language/5.0/_index.md:47]: I know I'm being picky and probably US-centric, @Pandrex247, but can you follow the mm/dd/yyyy convention, or spell out the Month?  My first reaction to this was to read this as Dec 05 and that would be difficult to predict... :-)  Thanks!

Comment (Pandrex247) [reply expression-language/5.0/_index.md]: Muscle memory 😜
I've clarified it

Comment (kwsutter) [reply expression-language/5.0/_index.md]: A bit more elaborate change than I expected, but this works...  ;-)  Thanks!

**Review by kwsutter — APPROVED** [review]
Thanks, @Pandrex247!

**Pull request merged**